terraform/modules/bap_env_gcp/variables.tf

```diff
@@ -49,9 +54,15 @@ variable "mariadb_machine_image" {
   default = "debian-cloud/debian-11"
 }
 
+variable "mariadb_disk_auto_delete" {
```

Comment:
Also, why don't we call this variable `persistent_disks` ? I think it's more clear. I know we need to negate this later on terraform but for the person configuring it, it will be easier to understand.